src/App.js

```diff
@@ -96,6 +97,7 @@ const App = () => (
               <UserRolesView />) : (<Redirect to="/Dashboard" />))}
           />
           <Route path="/accommodations" exact component={ViewAccomodation} />
+          <Route path="/bookings" exact component={bookAccommodation} />
```

Comment:
I think `/bookings/create` route is better. Then we shall `/bookings` routes to display all bookings.